### DISCUSSION TITLE
Adatok tárolása

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#40] [Kérdések és válaszok]
Felhőalapú e-pénztárgépekre vonatkozó műszaki követelmények között szerepel, hogy az alkalmazás rögzíti és az adómegállapításhoz való jog elévüléséig tárolja az epénztárgépen keletkező bizonylatokat. Az e-pénztárgéppel történő adatszolgáltatás esetén az adatok nyugtatárba történő beérkezésével, ezen adatok tekintetében, az adózót terhelő bizonylat megőrzési kötelezettség teljesül.

Milyen eszközön kell megvalósítani az adatok tárolását a felhőalapú pénztárgép esetén? Történhet az adatok tárolása egy központi szerveren (megfelelő titkosítás mellett)? 46.§ (2) szerint a nyugtatárba való beérkezéssel teljesül a tárolási kötelezettség. Ezek alapján elég addig az epg eszközön tárolni, amíg be nem érkezik a bizonylat a Nyugtatárba?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
